swagger.yaml

```diff
@@ -173,24 +289,42 @@ definitions:
         type: string
       attributes:
         $ref: "#/definitions/Attributes"
+      links:
+        $ref: "#/definitions/SelfLink"
+      relationships:
+        $ref: "#/definitions/Relationships"
 
-  Attributes:
+  ServicesResultObjects:
     properties:
-      name:
+      links:
+        $ref: "#/definitions/Links"
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/ServicesResourceObject"
+
+  ServicesResourceObject:
```

Comment:
You could go with links.related and link that to the locations object. I agree jsonapi isn't too clear though.